mypy/build.py

```diff
@@ -1149,11 +1164,11 @@ def validate_meta(meta: Optional[CacheMeta], id: str, path: Optional[str],
     fine_grained_cache = manager.use_fine_grained_cache()
 
     size = st.st_size
-    if size != meta.size and not fine_grained_cache:
+    if size != meta.size and not bazel and not fine_grained_cache:
```

Comment:
Why drop this check in bazel mode?

Reply:
Because in Bazel mode we don't do any of the cache validation things. If the cache file exists it should be used, period.

Reply:
Perhaps add a comment like "# Bazel ensures the cache is vlaid"?